EIPS/eip-2386.md

```diff
@@ -0,0 +1,208 @@
+---
+eip: 2386
+title: Walletstore
+author: Jim McDonald (Jim@mcdee.net)
+discussions-to: https://ethereum-magicians.org/t/eip-2386-walletstore/3792
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-11-21
+requires: 2333, 2334
+---
+
+## Simple Summary
+
+A JSON format for the storage and retrieval of Ethereum 2 wallet definitions.
+
+## Abstract
+
+Ethereum has the concept of keystores: pieces of data that define a key (see [EIP-2333](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2333.md) for details).  This adds the concept of walletstores: stores that define wallets and how keys in said wallets can be created.
```

Comment:
Can you change these links to the canonical format (https://eips.ethereum.org...) ?

Reply:
Updated; thanks.